_component_design/search.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Search Bar
+parent: Search and Filters
+layout: component
+category: Components
+usage: >
+  A search box is a combination of input field and submit button. One may think that the search box doesn’t need a design; after all, it’s just two simple elements. But since the search box is one of the most frequently used design element on content-heavy websites, its usability is critical.
+preview-image: preview-images/search.svg
+resource: true
+status: stable
+last-modified: 2017-11-07
+---
+
+{% include toc.html %}
+
+## Introduction
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+### When to use
+When an app supports large amounts of information, users should be able to quickly locate content by searching for it.
+
+### Best practices
+
+- Display the search element prominently on the page.
+- Always pair the search with the eyeglass icon.
+- Place search on every page that is displaying aggregate data sets.
+- Search bars are variable in size from app-to-app. To ensure smaller search bar function well for users, a 30 character minimum text input is displayed.
```

Comment:
remove hyphens
"The minimum text input size is 30 characters to ensure that smaller search bars are usable."

Reply:
updated